mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,209 @@
+
+In terms of these intersections, Waze recommends following the bow tie method
+
+![image](https://user-images.githubusercontent.com/13744156/34770951-d1db8876-f628-11e7-89f1-80308d36c53d.png)
+                                     *3(a)*
+
+Pros:
+
+- Allows greater control over u turns and left turns
+
+Cons:
+
+- Awkward appearance which causes the map to look bad
+- The angles will cause the routing engine to ask drivers to take `slight right` or `slight left` if they are not carefull constructed
+
+**OR**
+
+A divided road intersecting a 2-way road. A dual carriageway intersecting a bi-directional road can also be modelled this way 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34771066-40c4f6f0-f629-11e7-8692-c3582c86a9c5.png)
+                                      *3(b)*
+
+
+- Simplicity dictates that the mapper follows the **3(b)** form of intersection modelling.
+- Again there should not be any sausage road (braided street) type collapses in these cases. If you come across any, change it to this simpler intersection 👆 
+
+### Case 4: When two dual carriageways intersect each other: A divided road intersecting a divided road
+
+
+![image](https://user-images.githubusercontent.com/13744156/34771167-a05941de-f629-11e7-9a36-75f58fb8e124.png)
+                                     *4(a)*
+
+Make sure to change any sausage style collapses to a simple intersection type 👆 
+
+Wrong 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34771198-c12430cc-f629-11e7-8470-05e7fb47574e.png)
+*4(b) In these cases of "suasage" type collapses, make sure the intersection is remodelled back to 4(a)*
+
+### Examples of sausage type intersections and how to handle them
+
+#### Example 1
+
+ Wrong 👇 
+
+ ![image](https://user-images.githubusercontent.com/13744156/34771394-7cd72efa-f62a-11e7-8576-3da6c665ccf8.png)
+
+ Correct 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34771425-92cd333a-f62a-11e7-87f3-42866ac420c5.png)
+
+#### Example 2 
+
+Wrong 👇 
+
+
+![image](https://user-images.githubusercontent.com/13744156/34771482-b5078202-f62a-11e7-8cec-afd3b7f9b1d7.png)
+
+Correct 👇 
+
+
+![image](https://user-images.githubusercontent.com/13744156/34771511-cd8a0ac0-f62a-11e7-960c-f0efd53d13b8.png)
+
+#### Example 3 : 
+
+Wrong 👇  
+
+
+![image](https://user-images.githubusercontent.com/13744156/34771584-0a40ba54-f62b-11e7-86ff-14bb785a8ca1.png)
+
+Correct 👇 
+
+
+![image](https://user-images.githubusercontent.com/13744156/34771610-20ab4bec-f62b-11e7-928d-2fb2aa858841.png)
+
+#### Example 4
+
+Wrong 👇  
+
+![image](https://user-images.githubusercontent.com/13744156/34771634-3d40293a-f62b-11e7-8948-9647fcae90ea.png)
+
+Correct 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34771663-5bb60dd0-f62b-11e7-9cee-51cfdd5fdc6f.png)
+
+#### Example 5
+
+Wrong 👇 
+
+![image](https://user-images.githubusercontent.com/13744156/34771739-a89ea896-f62b-11e7-9032-03f4be3cf0ac.png)
```

Comment:
This approach is only incorrect because it implies a physical barrier between the turn lanes and opposing traffic, whereas the only separation is paint on the pavement.